.github/workflows/pre-commit-ubuntu-18.04.yaml

```diff
@@ -0,0 +1,279 @@
+name: Build OpenCV on Pull Requests
```

Comment:
A suggestoin: Can we make the name shorter so that CI messages on the PR page can be shorter and people can read much easier and locate the errrors on the modules faster?

Reply:
Yeah, sure, I'll do it.